components/header.js

```diff
@@ -1,16 +1,199 @@
+                    <li>{`9 ${icons['spades']}`}</li>
+                    <li>{`9 ${icons['clubs']}`}</li>
+                  </ul>
+                  <ul className={pageStyles.listHands}>
+                    <li>{`7 ${icons['spades']}`}</li>
+                    <li className={pageStyles.red}>{`7 ${icons['hearts']}`}</li>
+                    <li>{`7 ${icons['clubs']}`}</li>
+                    <li className={pageStyles.red}>{`7 ${icons['diamonds']}`}</li>
+                  </ul>
+                </div>
+              </li>
+              <li>
+                <h3>A pair of 2s can be beaten by a double sequence of 4+ pairs or a quartet</h3>
+                <div>
+                  <ul className={pageStyles.listHands}>
+                    <li>{`10 ${icons['spades']}`}</li>
+                    <li className={pageStyles.red}>{`10 ${icons['hearts']}`}</li>
+                    <li>{`J ${icons['clubs']}`}</li>
+                    <li>{`J ${icons['spades']}`}</li>
+                    <li>{`Q ${icons['spades']}`}</li>
+                    <li>{`Q ${icons['clubs']}`}</li>
+                    <li>{`K ${icons['spades']}`}</li>
+                    <li>{`K ${icons['clubs']}`}</li>
+                  </ul>
+                  <ul className={pageStyles.listHands}>
+                    <li>{`5 ${icons['spades']}`}</li>
+                    <li className={pageStyles.red}>{`5 ${icons['hearts']}`}</li>
+                    <li>{`5 ${icons['clubs']}`}</li>
+                    <li className={pageStyles.red}>{`5 ${icons['diamonds']}`}</li>
+                  </ul>
+                </div>
+              </li>
+              <li>
+                <h3>A triplet of 2s can only be beaten by a double sequence of 5+ pairs</h3>
+                <div>
+                  <ul className={pageStyles.listHands}>
+                    <li>{`3 ${icons['spades']}`}</li>
+                    <li className={pageStyles.red}>{`3 ${icons['hearts']}`}</li>
+                    <li>{`4 ${icons['clubs']}`}</li>
+                    <li>{`4 ${icons['spades']}`}</li>
+                    <li>{`5 ${icons['spades']}`}</li>
+                    <li>{`5 ${icons['clubs']}`}</li>
+                    <li>{`6 ${icons['spades']}`}</li>
+                    <li>{`6 ${icons['clubs']}`}</li>
+                    <li>{`7 ${icons['spades']}`}</li>
+                    <li>{`7 ${icons['clubs']}`}</li>
+                  </ul>
+                </div>
+              </li>
+            </ul>
+            <div>NOTE: Once a smash has been played, its combination type becomes what needs to be beaten by the next player.</div>
+          </div>
+          </div>
+          <div className={pageStyles.gameplay}>
+            <NavHeader handleClick={handleClick} rules={headerRules} type={'gameplay'} />
+            <div className={headerRules.gameplay ? `${pageStyles.listCardRank} ${pageStyles.ruleContent}` : pageStyles.hide}>
+              <ul className={pageStyles.listGameplay}>
```

Comment:
this format is a little hard to read, probably because of the many `<li>` element usages here. you can utilize more horizontal space here and add more padding between sections to let the text breathe. otherwise reading these rules feels difficult.

<img width="1039" alt="Screenshot 2025-04-11 at 6 39 09 PM" src="https://github.com/user-attachments/assets/c62226b1-6641-442f-8f95-50b1351270c6" />

Reply:
Gotcha, I'll try to make it easier to read!